src/renderer/store/game.store.ts

```diff
@@ -20,15 +21,18 @@ export const gameStore: {
     status: GameStatus;
     selectedGameVersion?: GameVersion;
     optionsMap?: Record<string, LuaOption & { section: string }>;
+    availableGameVersions: GameVersion[];
 } = reactive({
     isInitialized: false,
     status: GameStatus.CLOSED,
+    availableGameVersions: [],
 });
 
 async function refreshStore() {
     await db.gameVersions.clear();
     const installedVersions = await window.game.getInstalledVersions();
     await db.gameVersions.bulkPut(installedVersions);
+    gameStore.availableGameVersions = Array.from(installedVersions);
     const latestGameVersion = await db.gameVersions.orderBy("gameVersion").last();
```

Comment:
If we do this, we are actually using the `db.gameVersions` for anything useful? Can't we replace remaining usage in `DebugSidebar.vue` and `OfflineBattleComponent.vue` with this property and drop that db table entirely?

Reply:
Looking at how it's used, I definitely think it's not required to be in the DB. The one downside that comes to mind by dropping the DB is we lose the "key" of getting data out of the GameVersion type by finding it with a string. (e.g. ``db.gameVersions.get(version:string)``). We would have to search the array for the ``GameVersion.gameVersion`` value that matches the desired string.

As an alternative to the straight ``GameVersion[]`` in this PR, we could instead create a ``Map<string, GameVersion>`` to hold it keyed by version string. How do you feel about that?

Reply:
Sounds good to me, and `Map` has `.keys()` and `.values()` so it's easy to get what you need.